ui/pages/Project/selectors.js

```diff
@@ -769,8 +772,14 @@ export const getPageHeaderBreadcrumbIdSections = createSelector(
         content: `Family: ${family.displayName || ''}`,
         link: `/project/${project.projectGuid}/family_page/${family.familyGuid}`,
       }]
-      if (match.params.breadcrumbIdSection) {
-        breadcrumbIdSections.push({ content: snakecaseToTitlecase(match.params.breadcrumbIdSection), link: match.url })
+      const { breadcrumbIdSection } = match.params
+      if (breadcrumbIdSection) {
+        if (breadcrumbIdSection.startsWith('rnaseq_results/')) {
+          const individualId = individualsByGuid[breadcrumbIdSection.split('/')[1]]?.individualId || ''
```

Comment:
you should change the `Route` definition in the main `PageHeader` router to parse the params for you. Changing the definition to `/project/:projectGuid/:breadcrumb/:breadcrumbId?/:breadcrumbIdSection?/:breadcrumbIdSubsection*` would allow you to explicitly handle this case without needing to do all the string matching and parsing